### ISSUE TITLE
Way to set learning rate with multi-gpu training.

### ISSUE THREAD

**Issue by csj3684** [open-mmlab/mmdetection#10437]
Hi 

I  really appreciate your work.

I'm training TOOD with multi-gpu (https://download.openmmlab.com/mmdetection/v2.0/tood/tood_r50_fpn_1x_coco/tood_r50_fpn_1x_coco_20211210_103425.log)

they used 8 gpus with 2 batches per gpu (total 16 batches) with lr 0.01

and I'm using 

0. 4 gpus with 4 batches per gpu (lr 0.01)
1. 2 gpus with 4 batches per gpu (lr 0.005)
2. 1 gpu with 4 batches (lr 0.0025)

with 1 node.

They have the same ratio(total base batch size/ lr) but 4 gpus case has the most fluctuation > 2 gpus > 1 gpus

and also 4 gpus case is more fluctuating than official loss graph.

any right way or rule with setting lr to make 3 cases have the same(similar) fluctuations ?
